deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py

```diff
@@ -253,24 +253,24 @@ def check_vulnerability_scan_remove_alerts(wazuh_alerts_monitor, package, cve):
                                                          f"{package} is no longer vulnerable with {cve}")
 
 
-def check_provider_database_update_start_log(provider_name):
+def check_provider_database_update_start_log(provider_name, timeout=vd.VULN_DETECTOR_FAST_TIMEOUT):
     """Check the provider database update start event in ossec.log
 
     Args:
         provider_name (str): Provider name of the downloaded feed.
     """
-    check_vuln_detector_event(timeout=vd.VULN_DETECTOR_FAST_TIMEOUT,
+    check_vuln_detector_event(timeout=timeout,
                               callback=f"Starting '{provider_name}' database update",
                               error_message=f"Could not find {provider_name} update starting log")
 
 
-def check_provider_database_update_finish_log(provider_name):
+def check_provider_database_update_finish_log(provider_name, timeout=vd.VULN_DETECTOR_FAST_TIMEOUT):
```

Comment:
```suggestion
def check_provider_database_update_finish_log(provider_name, timeout=vd.VULN_DETECTOR_GLOBAL_TIMEOUT):
```

Reply:
This is not correct. You are modifying the generic timeout used for the rest of the test.

Be careful with this, because this can break other tests that use this function.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2547/commits/90bed077f7ebf72dd21e5d15b1af1bc674ba3704